pkg/cvo/updatepayload.go

```diff
@@ -225,6 +225,11 @@ func (r *payloadRetriever) fetchUpdatePayloadToDir(ctx context.Context, dir stri
 		Spec: batchv1.JobSpec{
 			ActiveDeadlineSeconds: deadline,
 			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						"openshift.io/required-scc": "privileged",
```

Comment:
> 	This is the most relaxed SCC and should be used only for cluster administration. Grant with caution [[1](https://docs.openshift.com/container-platform/4.17/authentication/managing-security-context-constraints.html#default-sccs_configuring-internal-oauth)]

I am not sure whether just using the `hostmount-anyuid` value with some modifications would be enough. However, other docs state that privileged pods must be used for `hostPath` volumes.

> A hostPath volume in an OpenShift Container Platform cluster mounts a file or directory from the host node’s filesystem into your pod. Most pods will not need a hostPath volume, but it does offer a quick option for testing should an application require it.	
> The cluster administrator must configure pods to run as privileged. This grants access to pods in the same node.  [[2](https://docs.openshift.com/container-platform/4.17/storage/persistent_storage/persistent_storage_local/persistent-storage-hostpath.html)]

Reply:
Using the hostmount-anyuid default SCC is not sufficient for the existing code
as the pod is not able to pass the SELinux permissions checks.
Additional SELinux, host file system, and/or code changes would be needed.